source/plugin/workspace/idea.rst

```diff
@@ -5,11 +5,50 @@ Setting Up IntelliJ IDEA
 This article describes how to configure your **IntelliJ IDEA** workspace for plugin development with SpongeAPI and
 :doc:`a build system such as Maven or Gradle <../buildsystem/>` or the `Minecraft Development plugin <https://minecraftdev.org/>`_.
 
-Gradle
-======
+If you want to create your project completely from scratch, please skip ahead to the Gradle or Maven sections.  Using the Minecraft Dev plugin sets up a working starting point and eliminates some of the guesswork in getting your project off the ground.
```

Comment:
Please try to keep the lines 120 characters or less. You have a few very long lines here. It's one of our Docs Guidelines (which can be found on the [SpongeDocs](https://docs.spongepowered.org/stable/en/contributing/spongedocs.html), for reference).